app/views/tickets/new.html.erb

```diff
@@ -4,64 +4,79 @@
   <%= render 'shared/breadcrumb' %>
 <% end%>
 
-<main role="main" id="content">
-  <div class="grid-row inner">
-    <div class="column-full">
+<main role="main" id="main-content" class="govuk-main-wrapper">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
       <% if @ticket.errors.any? %>
-        <div class="error-summary" role="alert" aria-labelledby="error-summary-heading">
-          <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
-            <%= t('.problem')%>
-          </h2>
-          <ul class="error-summary-list">
-            <% @ticket.errors.messages.each do |attr, error| %>
-              <li><a href="#error-<%= attr.to_s %>"> <%= error[0] %></a></li>
-            <% end %>
-          </ul>
-        </div>
+        <%
+          errors = []
+
+          @ticket.errors.messages.each do |attr, error|
+            errors << {
+              text: error[0],
+              href: "#error-#{attr.to_s}"
+            }
+          end
```

Comment:
Could this can use `map` instead of setting a variable and using `each`?

```suggestion
          errors = @ticket.errors.messages.map do |attr, error|
            {
              text: error[0],
              href: "#error-#{attr.to_s}"
             }
          end
```